IdentityCore/src/oauth2/token/MSIDBaseToken.h

```diff
@@ -56,6 +56,8 @@
 @property (readwrite) NSString *clientId;
 @property (readwrite) NSDictionary *additionalServerInfo;
 @property (readwrite) MSIDAccountIdentifier *accountIdentifier;
+@property (readwrite) NSDate *extendedExpiresOn;
```

Comment:
I don't think this one is applicable to base token. it should be in access token only similarly to expiresOn.

Reply:
updated